### PR TITLE
Feature: part labels

### PR DESCRIPTION
**Pull request by riedde**
Add labels for parts in measureBasedView.

### CODE REVIEW

**Comment by riedde** [comment]
Co-authored by: @nikobeer

File: add/data/xql/getMeasures.xql [hunk 1]
```diff
@@ -64,14 +64,26 @@ declare function local:getMeasures($mei as node(), $mdivID as xs:string) as xs:s
...
                                         for $measure in $part//mei:measure[@n = $measureN][1] | $measures[ancestor::mei:part = $part]
+                                            let $voiceRef := $part//mei:staffDef/@decls
+                                            let $voiceID := substring-after($voiceRef, '#')
                                             return
-                                                concat('{id:"', $measure/@xml:id, '", voice: "', $part//mei:staffDef/@decls, '"}')
+                                                concat('{id:"',
+                                                $measure/@xml:id,
+                                                '", voice: "', $voiceRef,
+                                                '", partLabel: "', $mei/id($voiceID)/@label,
+                                                '"}')
```
Comment (bwbohl) [inline add/data/xql/getMeasures.xql:86]: why is the content of the if and else clause exactly the same?

Comment (riedde) [reply add/data/xql/getMeasures.xql:86]: It isn't, the `then` is for processing `@label` and the `else` is for `@n`. I just have added the `partLabel`, but this should be switched into a function in the next release

Comment (bwbohl) [reply add/data/xql/getMeasures.xql:86]: sorry, my bad

**Review by bwbohl — APPROVED** [review]

**Pull request merged**